### PR TITLE
BaSP-MR-78: add/edit forms

### PR DESCRIPTION
**Pull request by flordimonaco**
**Description**
My [Trello's ticket](https://trello.com/c/BBEoGXc4/78-subscription-add-edit-form) said that I should do conditional rendering based on a momentary state.

_All items on the checklist have been resolved._

### CODE REVIEW

File: src/Components/Subscriptions/Table/TableSubscription.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        `${process.env.REACT_APP_API_URL}/subscriptions/${id}`,
+        {
+          method: 'PUT',
+          headers: {
+            'Content-type': 'application/json'
+          },
+          body: JSON.stringify(editSubscriptions)
+        }
+      );
+      console.log(subscriptionUpdated);
```
Comment (CristianLotorto) [inline src/Components/Subscriptions/Table/TableSubscription.jsx:29]: this console.log should not be here

File: src/Components/Subscriptions/Table/TableSubscription.jsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    const findSubscription = subscription.find((sub) => sub._id === id);
+    editSubscriptionDB(findSubscription._id, editSubscriptions);
+    findIdEdit(id);
+  };
+
+  return (
+    <>
+      {table && (
+        <>
+          <button
```
Comment (CristianLotorto) [inline src/Components/Subscriptions/Table/TableSubscription.jsx:56]: methods on this button triggers an infinite loop

**Review by CristianLotorto — CHANGES_REQUESTED** [review]
take a look at the comments

File: src/Components/Subscriptions/Table/TableSubscription.jsx (hunk 1 above)
Comment (NicoCagnina) [reply src/Components/Subscriptions/Table/TableSubscription.jsx:29]: remove this console log

**Review by NicoCagnina — CHANGES_REQUESTED** [review]
please add a description

File: src/Components/Subscriptions/Table/Index.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  };
+
+  const deleteSubscription = async (id) => {
+    await deleteSubscriptionDB(id);
+    setSubscription([...subscription.filter((sub) => sub._id !== id)]);
+  };
+
+  return (
+    <section className={style.containerTable}>
+      <TableSubscriptions
```
Comment (FrancoLelli) [inline src/Components/Subscriptions/Table/Index.jsx:42]:
```suggestion
            {!subscription.length ? (
        <p>No active Subscriptions</p>
      ) : (
        <TableSubscriptions  subscription={subscription} setSubscription={setSubscription} deleteSubscription{deleteSubscription}/>
      )}
```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]
Add validation. So, we render a text in case there are no active subscriptions. Good Job!

**Review by FrancoLelli — APPROVED** [review]

**Review by CristianLotorto — APPROVED** [review]

**Review by cbaleani — APPROVED** [review]
Looks good!

**Pull request merged**